radio/src/targets/flysky/AFHDS2A_a7105.cpp

```diff
@@ -17,11 +17,28 @@
 #include "iface_a7105.h"
 #include "opentx.h"
 
-#define AFHDS2A_TXPACKET_SIZE 37
-#define AFHDS2A_RXPACKET_SIZE 37
 #define AFHDS2A_HUB_TELEMETRY
 //#define AFHDS2A_NUMFREQ			16
 
+#if ((AFHDS2A_CHANNELS - 0) > 18)
+# error "wrong number of channels"
+#endif
+
+#if ((AFHDS2A_CHANNELS - 0) > 16)
+# ifdef AFHDS2A_LQI_CH 
+#  if ((AFHDS2A_LQI_CH > AFHDS2A_CHANNELS) || (AFHDS2A_LQI_CH <= 16))
+#   warning "wrong AFHDS2A_LQI_CH"
+#   ifdef AFHDS2A_LQI_CH
+#    undef AFHDS2A_LQI_CH
+#   endif
+#  endif
+# endif
+#else 
```

Comment:
if 16 or less channels then remove AFHDS2A_LQI_CH definition. Should leave it alone to allow previous config of LQI  at ch14.